### DISCUSSION TITLE
Time Zone Localisation options

### DISCUSSION THREAD

**Discussion by dunc1ca** [immich-app/immich#14398] [Feature Request]
### The bug

I live in Canada and speak English and use the date format (yyyy-mm-dd).  Like a significant but growing minority of Canadians, we use the 24-hour clock and only use am and pm in everyday speech.  Can there be an option for English yyyy-mm-dd HH:mm:ss please?

### The OS that Immich Server is running on

Windows Docker

### Version of Immich Server

v1.121.0

### Version of Immich Mobile App

n/a

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
n/a
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

1. localhost:2283
2. Account Settings
3. App Settings
4. Custom Locale
...

**Comment by bo0tzz** [comment]
This is already supported with the "default locale" option.

![image](https://github.com/user-attachments/assets/5bbb359c-1702-4808-b568-bf6439dca1ef)
